pipelines/coaddDiffMatchedQualityExtended.yaml

```diff
@@ -1,56 +1,68 @@
 description: |
   Matched difference (measured vs reference) plots/metrics
+parameters:
+  mag_x_min: 17
+  mag_x_max: 29
+  mag_diff_min: -1000
+  mag_diff_max: 1000
+  mag_chi_min: -5
+  mag_chi_max: 5
+  pos_diff_min: -100
+  pos_diff_max: 100
+  pos_chi_min: -5
+  pos_chi_max: 5
 tasks:
   diff_matched_analysis:
     class: lsst.analysis.tools.tasks.DiffMatchedAnalysisTask
     config:
       connections.outputName: diff_matched_truth_summary_objectTable_tract
 
       # plots
-      atools.matchedRefCModelMagDiff: MatchedRefCoaddDiffMagPlot
+      atools.matchedRefCModelMagDiff: MatchedRefCoaddDiffMagTool
+      # Doesn't work; interpreted as float
+      # atools.matchedRefCModelMagDiff.produce.plot.xLims: [parameters.mag_x_min, parameters.mag_x_max]
```

Comment:
Does this need to be here for any particular historical reason?

Reply:
Sort of; it's a note that this doesn't work even though it should. In this case I ended up preferring to define `lims_mag_x` in the python block. Anyway, I'm going to address this on a subsequent ticket ([DM-41049](https://jira.lsstcorp.org/browse/DM-41049)).